templates/js/copilot-plugin-from-scratch-api-key/package.json.tpl

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "{{SafeProjectNameLowerCase}}",
+    "version": "1.0.0",
+    "scripts": {
+        "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run dev",
+        "dev": "func start --javascript --language-worker=\"--inspect=9229\" --port \"7071\" --cors \"*\"",
+        "start": "npx func start",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "key:gen": "node ./keygen.js",
```

Comment:
Is there any reason to add :, why not keygen for task name?

Reply:
Updated